Sources/AppcuesKit/Presentation/Debugger/FloatingView/DebugView.swift

```diff
@@ -441,6 +467,23 @@ internal class DebugView: UIView {
             completion(true)
         }
     }
+
+    private func animateToast(visible isVisible: Bool, animated: Bool, completion: (() -> Void)?) {
+
+        let animations: () -> Void = {
+            self.toastWrapperView.alpha = isVisible ? 1 : 0
+        }
+
+        if animated {
+            UIView.animate(
+                withDuration: 0.6,
+                animations: animations) { _ in completion?() }
+        } else {
+            animations()
+            completion?()
+        }
+    }
```

Comment:
If we ever have a more general need for toast capabilities, perhaps this functionality gets hoisted up elsewhere and generalized. For now, I decided to keep it localized and specific to the one and only known use case here.